multisig/proposals/ethereum/vip-026/index.ts

```diff
@@ -0,0 +1,145 @@
+export const FRAX_INITIAL_SUPPLY = parseUnits("5000", 18);
+export const sFRAX_INITIAL_SUPPLY = parseUnits("4800", 18);
+export const CHAINLINK_FRAX_FEED = "0xB9E1E3A9feFf48998E45Fa90847ed4D467E8BcfD";
+
+export const vip026 = () => {
+  return makeProposal([
+    // Configure Oracle
+    {
+      target: ethereum.CHAINLINK_ORACLE,
+      signature: "setTokenConfig((address,address,uint256))",
+      params: [[FRAX, CHAINLINK_FRAX_FEED, 3600]],
+    },
+    {
+      target: ethereum.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [
+          FRAX,
+          [ethereum.CHAINLINK_ORACLE, ethers.constants.AddressZero, ethers.constants.AddressZero],
+          [true, false, false],
+        ],
+      ],
+    },
+    {
+      target: ethereum.RESILIENT_ORACLE,
+      signature: "setTokenConfig((address,address[3],bool[3]))",
+      params: [
+        [sFRAX, [sFRAXOracle, ethers.constants.AddressZero, ethers.constants.AddressZero], [true, false, false]],
+      ],
+    },
+
+    // Add FRAX Market
+    {
+      target: VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [FRAX, FRAX_INITIAL_SUPPLY, ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: FRAX,
+      signature: "approve(address,uint256)",
+      params: [ethereum.POOL_REGISTRY, 0],
+    },
+    {
+      target: FRAX,
+      signature: "approve(address,uint256)",
+      params: [ethereum.POOL_REGISTRY, FRAX_INITIAL_SUPPLY],
+    },
+    {
+      target: vFRAX,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["7200"],
+    },
+    {
+      target: ethereum.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          vFRAX,
+          "750000000000000000",
+          "800000000000000000",
+          "5000000000000000000000",
+          VTREASURY,
+          "10000000000000000000000000",
+          "8000000000000000000000000",
+        ],
+      ],
+    },
+
+    // Add sFRAX Market
+    {
+      target: VTREASURY,
+      signature: "withdrawTreasuryToken(address,uint256,address)",
+      params: [sFRAX, sFRAX_INITIAL_SUPPLY, ethereum.NORMAL_TIMELOCK],
+    },
+    {
+      target: sFRAX,
+      signature: "approve(address,uint256)",
+      params: [ethereum.POOL_REGISTRY, 0],
+    },
+    {
+      target: sFRAX,
+      signature: "approve(address,uint256)",
+      params: [ethereum.POOL_REGISTRY, sFRAX_INITIAL_SUPPLY],
+    },
+    {
+      target: vsFRAX,
+      signature: "setReduceReservesBlockDelta(uint256)",
+      params: ["7200"],
+    },
+    {
+      target: ethereum.POOL_REGISTRY,
+      signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+      params: [
+        [
+          vsFRAX,
+          "750000000000000000",
+          "800000000000000000",
+          "4800000000000000000000",
+          VTREASURY,
```

Comment:
The receiver of the minted vTokens should be `0x6e74053a3798e0fC9a9775F7995316b27f21c4D2`